ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/operations/Attestation.java

```diff
@@ -53,6 +53,13 @@ public Attestation(
     this.signature = signature;
   }
 
+  public Attestation(Attestation attestation) {
+    this.aggregation_bits = attestation.getAggregation_bits().copy();
+    this.data = new AttestationData(attestation.getData());
+    this.custody_bitfield = attestation.getCustody_bitfield().copy();
+    this.signature = BLSSignature.fromBytes(attestation.getAggregate_signature().toBytes());
```

Comment:
Do we need to `copy()` the signature bytes here?
```suggestion
    this.signature = BLSSignature.fromBytes(attestation.getAggregate_signature().toBytes().copy());
```

Reply:
from Bytes is creating a new signature here so it should be fine. nice check tho